services/_shared/llm_client.py

```
@@ -885,6 +884,7 @@ def review_diff(
     file_contents: dict[str, str] | None = None,
     cross_file_contents: dict[str, str] | None = None,
     runtime_context: str | None = None,
+    voice: VoiceSelection = "caveman",
```

Ruff (0.15.20)

[error] Undefined name `VoiceSelection`

(F821)